applications/StructuralMechanicsApplication/custom_conditions/surface_load_condition_3d.cpp

```diff
@@ -328,4 +328,27 @@ namespace Kratos
         KRATOS_CATCH("")
     }
 
+    //************************************************************************************
+    //************************************************************************************
+
+    std::string SurfaceLoadCondition3D::Info() const
+    {
+        std::string condition_name = "PointLoadCondition";
+        const unsigned int number_of_nodes = GetGeometry().size();
+
```

Comment:
Huuuum, I thing that there is a method name for this pourpose

Reply:
Name

Reply:
I could not find the Name method in the element/condition base. Am I looking at the wrong place?

Reply:
True, I checked wrongly, sorry